### PR TITLE
publish version 6.114 of the spec, first v6 published version

### PR DESCRIPTION
**Pull request by BryceButenhof**
publish version 6.114 of the documentation

### CODE REVIEW

File: guides/version-support-matrix.markdown [hunk 1]
```diff
@@ -35,11 +36,27 @@ release of Metasys.
...
+- Time Series
+  - Get trended object (`/timeSeries/{objectId}`)
+  - Get trended attribute (`/timeSeries/{objectId}/trendedAttributes/{attributeId}`)
+
+#### Deprecated Operations
+
+- Alarms  
+  - Get alarms - (`GET /alarms`)
+- Audits
+  - Get audits - (`GET /audits`)
```
Comment (michaelgwelch) [inline guides/version-support-matrix.markdown:58]: So looks like the change log for v5 is wrong. These two operations were deprecated in v5. They are completely missing in v6. So I'd move this message down to v5 section and in this section list them as removed operations.

Comment (michaelgwelch) [reply guides/version-support-matrix.markdown:58]: In the removal note it'd be nice to mention the replace which is `/activites?activityType=`

**Comment by michaelgwelch** [comment]
Also, this is minor but both v5 and v6 have this note:

image


Which is attempting to mimic a note found on v4. But only the v4 link is useful because it links back to our legacy v3 documentation which includes a version picker. So it does indeed link back to v3 and older.

but the v6 and v5 links back to v4 just link to v4. There's no way to navigate to v1-v3. So I'd say rather than worrying about fixing this note upstream and then moving it to this PR. We'll just add a task to the back log to fix that link to just take you back to the landing page.

File: guides/version-support-matrix.markdown (hunk 1 above)
Comment (BryceButenhof) [reply guides/version-support-matrix.markdown]: Fixed

**Comment by BryceButenhof** [comment]
> Also, this is minor but both v5 and v6 have this note:
> 
> image
> Which is attempting to mimic a note found on v4. But only the v4 link is useful because it links back to our legacy v3 documentation which includes a version picker. So it does indeed link back to v3 and older.
> 
> but the v6 and v5 links back to v4 just link to v4. There's no way to navigate to v1-v3. So I'd say rather than worrying about fixing this note upstream and then moving it to this PR. We'll just add a task to the back log to fix that link to just take you back to the landing page. Sometime in the future when we re-publish it'll be fixed then

I will add a task to team Midnight backlog

**Review by michaelgwelch — APPROVED** [review]

**Pull request merged**